packages/bsky/tests/notification-server.test.ts

```diff
@@ -0,0 +1,74 @@
+import AtpAgent, { AtUri } from '@atproto/api'
+import { TestNetwork } from '@atproto/dev-env'
+import { SeedClient } from './seeds/client'
+import basicSeed from './seeds/basic'
+import { NotificationServer } from '../src/notifications'
+import { Database } from '../src'
+import { sql } from 'kysely'
+
+describe('notification views', () => {
```

Comment:
it'd be cool to see some tests of it actually sending the notifications

you could wire up a very simple mock gorush service here (just an express app with one route) that makes sure that the notifs get sent properly formatted

Reply:
I added a test to check the notif format!

Reply:
Let me know if you still want me to mock `gorush` but it really doesn't return anything useful. Just successes every time.